src/equations/compressible_euler_1d.jl

```diff
@@ -134,7 +134,7 @@ function initial_condition_density_wave(x, t, equations::CompressibleEulerEquati
     rho = 1 + 0.98 * sinpi(2 * (x[1] - t * v1))
     rho_v1 = rho * v1
     p = 20
-    rho_e = p / (equations.gamma - 1) + 1 / 2 * rho * v1^2
+    rho_e = p / (equations.gamma - 1) + 0.5 * rho * v1^2
```

Comment:
```suggestion
    rho_e = p / (equations.gamma - 1) + 0.5f0 * rho * v1^2
```

Reply:
Sorry but I was advised not to change number types from the initial condition and source terms. If you change your mind, please inform me. If not, I could further leave some comments on these functions if you want.

Reply:
Well, these lines were changed in the PR - that's why I saw them in the review. Since these changes belong to the case where it's clear what to do, it's fine to change them here.

Reply:
It's not clear here. Whether we change the `0.5` to `0.5f0` or not, the result always returns a static vector of `Float64` numbers. So it needs further discussion to make it possible to return `Float32` numbers.

Reply:
> Whether we change the 0.5 to 0.5f0 or not, the result always returns a static vector of Float64 numbers. So it needs further discussion to make it possible to return Float32 numbers.

That's true. I just meant it's fine to change `1 / 2` to `0.5f0` in cases like these.